src/lib/scheduler/node_queue_scheduler.cpp

```diff
@@ -209,4 +219,41 @@ void NodeQueueScheduler::_group_tasks(const std::vector<std::shared_ptr<Abstract
   }
 }
 
+// with 240 groups.................
+void NodeQueueScheduler::_group_numa_aware(const std::vector<std::shared_ptr<AbstractTask>>& tasks) const {
+  auto round_robin_counter = std::vector<int>(_queue_count, 0);
+
+  // std::vector<std::shared_ptr<AbstractTask>> grouped_tasks(_workers_per_node * _queue_count);
+  std::vector<std::shared_ptr<AbstractTask>> grouped_tasks(_group_number_per_node * _queue_count);
```

Comment:
What is a "group number"? Additionally, it should probably not be a member variable, but instead a constant